exaudfclient/base/.bazelrc

```diff
@@ -8,7 +8,14 @@ build:fast-binary-py3 --copt='-DCUSTOM_LIBEXAUDFLIB_PATH="/exaudf/libexaudflib_c
 build:fast-binary-both --config=fast-binary-py2 --config=fast-binary-py3
 build:slow-wrapper-py2 --define binary_type=slow_wrapper //:exaudfclient
 build:slow-wrapper-py3 --define binary_type=slow_wrapper //:exaudfclient_py3
-build:slow-wrapper-both --config=slow-wrapper-py2 --config=slow-wrapper-py3 
+build:slow-wrapper-both --config=slow-wrapper-py2 --config=slow-wrapper-py3
+build:static-binary-py3 //:exaudfclient_py3_static
+build:static-binary-py2 //:exaudfclient_py2_static
+#Static binary uses the same executable as the normal exaudflclient (with dynamic linking)
```

Comment:
Not sure if understand this comment

Reply:
I believe this comment is an artifact from a previous try. I removed it.

Reply:
Done